### PR TITLE
Criterion runner

### PR DESCRIPTION
**Pull request by falkecarlsen**
Works just like CGAAL benching; do `cargo bench` in criterion-runner to run all benches.
Those which fails because of Prism are commented out. 

We should ensure that the recorded benches are those which we are interested in, e.g. not all mexi combinations are used, which do we want/need?

### CODE REVIEW

File: benchmarking/mexican-standoff/mexican-3-players-2-lives.prism [hunk 1]
```diff
@@ -38,31 +38,31 @@ module shooter2 = shooter1 [health1 = health2, health2 = health1, p1wait = p2wai
...
+  [p1wait,p2shoot1,p3wait] true -> (health1' = (health1-1));
+  [p1wait,p2wait,p3shoot1] true -> (health1' = (health1-1));
+  [p1wait,p2wait,p3shoot2] true -> (health2' = (health2-1));
+  [p1wait,p2wait,p3wait] true -> true;
+  [p1wait,p2shoot3,p3shoot1] true -> (health1' = (health1-1)) & (health3' = (health3-1));
+  [p1wait,p2shoot3,p3shoot2] true -> (health2' = (health2-1)) & (health3' = (health3-1));
+  [p1wait,p2shoot3,p3wait] true -> (health3' = (health3-1));
+  [p1shoot2,p2shoot1,p3shoot1] true -> (health1' = (health1-2)) & (health2' = (health2-1));
+  [p1shoot2,p2shoot1,p3shoot2] true -> (health1' = (health1-1)) & (health2' = (health2-2));
+  [p1shoot2,p2shoot1,p3wait] true -> (health1' = (health1-1)) & (health2' = (health2-1));
```
Comment (NicEastvillage) [inline benchmarking/mexican-standoff/mexican-3-players-2-lives.prism]: Why were the `max`s removed here @larsbpf ?

Comment (larsbpf) [reply benchmarking/mexican-standoff/mexican-3-players-2-lives.prism]: I added a range to health instead, from [0..max_health]

Comment (NicEastvillage) [reply benchmarking/mexican-standoff/mexican-3-players-2-lives.prism]: Okay. Hopefully it won't crash if then the variable gets out of bounds (like in CGAAL)

**Review by NicEastvillage — APPROVED** [review]

**Comment by larsbpf** [comment]
Solves #10 and #9

**Comment by NicEastvillage** [comment]
Ready for merge?

**Comment by larsbopark** [comment]
For some reason does PRISM evaluate 2 RC queries as the opposite as expected(probably because of homebrewed model). I don't have time to fix it today.

**Pull request merged**